src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html

```diff
@@ -3,16 +3,19 @@
     <div>
       <span class="f8-space-masthead-name">{{space?.attributes.name | spaceName}}</span> <span class="f8-space-masthead-owner">bobross_925</span>
     </div>
-    <div>
-      <span class="pficon pficon-users fa-lg"></span> 59
+    <span class="pficon pficon-users fa-lg"></span> 59
+    <ng-container *ngIf='userOwnsSpace; else userDoesNotOwnSpaceHeaderInfo'>
       <span class="f8-space-masthead-vertical-divider-collab"></span>
       <span class="pficon pficon-add-circle-o fa-lg"></span> Add Collaborator
-    </div>
+    </ng-container>
+    <ng-template #userDoesNotOwnSpaceHeaderInfo>
+      <span id="space-description-collaborators-placeholder">Collaborators</span>
+    </ng-template>
   </div>
-  <div class="f8-space-masthead-vertical-divider"></div>
+  <div *ngIf="userOwnsSpace" class="f8-space-masthead-vertical-divider"></div>
```

Comment:
I believe we should keep the divider considering we're still displaying the description -- just not allowing editing.